js/languages/en-US.json

```diff
@@ -526,7 +527,9 @@
     "smtpServerSuccess": "SMTP Connection Success",
     "noSMTPConnection": "The connection to the SMTP server failed.",
     "badSMTPAuthentication": "The username/password combination used failed.",
-    "goodSMTPAuthentication": "You have successfully connected to the SMTP server."
+    "goodSMTPAuthentication": "You have successfully connected to the SMTP server.",
+    "tagIsTooLongHeadline": "The tag is too long",
+    "tagIsTooLongBody": "Tags cannot be more than 40 characters long."
```

Comment:
Would it be worth it to variabalize the max length here? Otherwise, if we ever change it, the translations will have the old amount.